content/apps/django.md

```diff
@@ -40,6 +38,8 @@ dj-database-url==0.3.0 #grabs environment variables and dumps them into a Django
 waitress==0.8.9 #a pure python WSGI server that is a replacement for gunicorn
 ```
 
+Note that [you will need to vendor the dependencies](http://docs.cloudfoundry.org/buildpacks/python/#vendoring) before deploying.
```

Comment:
The buildpack will include pypi dependencies, but if you have things in your `requirements.txt` like:

```
git+https://github.com/JoshData/commonmark-py-plaintext
git+https://github.com/if-then-fund/django-html-emailer
```

then you will  need to run `pip download`